[2.8.6] Doc how to pin an unpinned webhook (#1396)
rancher/rancher-docs: versioned_docs/version-2.8/reference-guides/rancher-webhook.md

To verify if the webhook is pinned, bring up the Rancher kubectl shell, or switch to a terminal session, and run:

```bash
kubectl get settings rancher-webhook-version
```

If the webhook is pinned, you'll see output with a `VALUE` field that matches the `Webhook Version` from the above table:

```text
NAME                       VALUE
rancher-webhook-version    103.0.2+up0.4.3
```

If the webhook is unpinned, the `VALUE` column will be blank.

There are two ways to pin the webhook in Helm installations. If you're running Rancher v2.8.3 and using a "values" YAML file (typically called `values.yaml`), add this block to the file:

```yaml
extraEnv:
  - name: CATTLE_RANCHER_WEBHOOK_VERSION
    value: 103.0.2+up0.4.3
```

Then, run the command:

```bash
helm upgrade --install rancher rancher-latest/rancher --namespace cattle-system --reuse-values --values PATH/TO/values.yaml
```

You can instead specify the webhook version directly on the command-line:

```bash
helm upgrade --install rancher rancher-latest/rancher --namespace cattle-system --reuse-values \
    --set extraEnv[0].name=CATTLE_RANCHER_WEBHOOK_VERSION \
    --set extraEnv[0].value=103.0.2+up0.4.3
```

> After doing this, the webhook field in the UI should be the value specified in the `helm` command, and the above `kubectl get settings` command should have the same value in the `VALUE` column.

Reviewer: ```suggestion
As a result, the webhook field in the UI should have the value specified in the `helm` command, and the above `kubectl get settings` command should have the same value in the `VALUE` column.
```
